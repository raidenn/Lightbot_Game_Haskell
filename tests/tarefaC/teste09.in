Abcdef
zzzzzg
abcdeF
0 0 E
SSSSSLESSESSSSSL
