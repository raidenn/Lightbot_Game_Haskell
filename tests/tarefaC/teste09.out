5 0
0 2
FIM 16
